scripts/scripts.js

```diff
@@ -90,9 +90,65 @@ sampleRUM.observe = ((elements) => {
   });
 });
 
+sampleRUM.targetselector = (element) => {
+  let selector = element.tagName.toLowerCase();
+  // always include these attributes as they have valuable info
+  const distinctAttributes = ['src', 'href'];
+  // capture the actual click target instead of a child element
+  const terminateearly = (e) => e.tagName.toLowerCase() === 'a'
+    || e.tagName.toLowerCase() === 'button';
+  if (element === document.body || element === document.documentElement) {
+    return selector;
+  }
+  if (element.id) {
+    return `#${element.id}`;
```

Comment:
If there is a unique `#id` attribute on the tag, we just use that, i.e. `#adobe-max--the-creativity-conference`